tasks/br/data.py

```diff
@@ -231,7 +231,7 @@ def columns(self):
         filepath = "meta/{tablename}.csv".format(tablename=self.tablename)
 
         session = current_session()
-        with open(os.path.join(os.path.dirname(__file__),filepath)) as tsvfile:
+        with open(os.path.join(os.path.dirname(__file__), filepath), encoding='latin1') as tsvfile:
```

Comment:
Is mandatory to include the encoding? In python2 the default is latin1 and now is utf-8 in p3?

Reply:
This is a bit bold, but I think for the better. In py2, this returned undecoded strings (byte arrays), while in py3, it decodes it to utf-8. If you see a bit below, there was a bit of code to decode as latin1 the name field. In py3, the best way to do this is to pass the encoding to use for the whole file, otherwise you get decoding errors while opening it.

Reply:
Ok, looks good to me too